test/simple/StatsPrinter.h

```diff
@@ -24,5 +24,8 @@ class StatsPrinter : public Stats {
   void frameWritten(const std::string& frameType) override;
   void frameRead(const std::string& frameType) override;
   void resumeBufferChanged(int framesCountDelta, int dataSizeDelta) override;
+
+  void connectionClosedInServerConnectionAcceptor(
```

Comment:
I think this should be very specific because it is used for one specific case. I believe we should have way more specific counters all over the library. Essentially the idea is that for debugging reasons you want to know if you library is hitting any edge cases in production, if you are refusing connections / closing / rejecting frames and such. Would you agree with that statement?